WolmoReactiveCore/Action.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Action.swift
+//  WolmoReactiveCore
+//
+//  Created by Diego Quiros on 22/11/2018.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+import MBProgressHUD
+import ReactiveSwift
+
+extension Action {
+    
+    /*
```

Comment:
It's just a detail but can we use `/**    */` instead of `/*     **/` as in the other files?

Reply:
sounds good